quiz/settings.py

```diff
@@ -45,9 +45,7 @@
 )
 
 REST_FRAMEWORK = {
-    'DEFAULT_PERMISSION_CLASSES': [
-        'rest_framework.permissions.IsAdminUser'
-    ],
```

Comment:
This is fine, for now. I would rather get functional tests in place, over REST authentication. Can you create another task to track test permissions?

Reply:
Sure, I'll create one in my name!